### ISSUE TITLE
UTF-8 charaters on Windows

### ISSUE THREAD

**Issue by msroth** [ejlb/google-open-image-download#1]
Thanks for this script! I am running it on Windows and it seems to dump out when it hits some of the foreign characters in the description (?) column.  Consequently, I am only able to download about 90 image files.  Any way to make it decode those characters?  Or this there another issue?

**Comment by ejlb** [comment]
Hi @msroth thanks for the issue. I think this is due to csv dict reader. I'm going to push something now that should fix it.

**Comment by msroth** [comment]
Thank you.  I’ll look for it.

Cheers,
Scott Roth
[Armedia_blue_small]
Test & Evaluation Lead
Science and Technology Integration Lab (STIL)
2070 Chain Bridge Dr., #100
Vienna, VA  22182
scott.roth@armedia.com
Cell: 703-408-1187
Office: 571-395-8875

**Comment by ejlb** [comment]
@msroth could you try again when you get a chance. I can't reproduce the issue but I think this should fix it.

**Comment by msroth** [comment]
I just tried and get a new error.  I posted it on GitHub.  Something about dict has no iteritems attribute.  What version python?  I’m using 3.5.2.  I’m also on Windows.  I’ve seen a difference between Windows and Linux when it comes to UTF-8 chars.

Cheers,
Scott Roth
[Armedia_blue_small]
Test & Evaluation Lead
Science and Technology Integration Lab (STIL)
2070 Chain Bridge Dr., #100
Vienna, VA  22182
scott.roth@armedia.com
Cell: 703-408-1187
Office: 571-395-8875
